components/Input/Input.tsx

```diff
@@ -0,0 +1,81 @@
+import { InputHTMLAttributes } from "react";
+
+interface InputProps
+  extends Pick<
+    InputHTMLAttributes<HTMLInputElement>,
+    | "id"
+    | "type"
+    | "value"
+    | "onChange"
+    | "name"
+    | "required"
+    | "className"
+    | "disabled"
+  > {
+  error?: boolean;
+  label: string;
+  helperText: string;
+}
+
+const Input = ({
+  id,
+  error = false,
+  label,
+  required,
+  helperText,
+  disabled,
+  className,
+  ...props
+}: InputProps) => {
+  const spanStyles = `font-medium group-focus-within:font-bold ${
+    error ? "text-red font-bold" : ""
+  }`;
+  const inputStyles = `${className} w-full h-[46px] rounded-[3px] p-2 ring-1 mt-2
+  focus-visible:ring-[3px] outline-none ring-inset focus-visible:shadow-2md focus-visible:bg-blue-light active:ring-orange 
+  ${disabled ? "ring-gray-dark" : ""} ${
+    error ? "ring-3 ring-red font-bold" : "ring-blue"
+  }`;
+  const svgStyles = `absolute right-8 top-16 ${error ? "block" : "hidden"}`;
+
+  return (
+    <div className="text-base font-inter space-y-1 group">
+      <label htmlFor={id}>
+        <span className={spanStyles}>{`${required ? "*" : ""}${label}`}</span>
```

Comment:
Aca no hace falta el template string

```suggestion
        <span className={spanStyles}>{required ? "*" : ""} {label}</span>
```